packages/builder/src/builderStore/store/screenTemplates/emptyContainerScreen.js

```diff
@@ -0,0 +1,22 @@
+export default {
```

Comment:
Do we need this concept? The "Empty X screen"? It seems a bit weird and doesn't really add much

Reply:
Hmm. So, if you want to build a screen yousself, from scratch, there are the 3 "Screen Types" - the field that removed completely (aka Layout Component, aka Base Component)

These 3 screen types are
- Row Detail - loads a row into context from a `/something/:id` url
- New Row - creates a row in context, and loads the model
- Container - anything that falls outside of the above 2 etc... e.g. Lists of stuff, or screens where components do all the work (e.g. table)

"emptyContainerScreen" is the last type here - I think we still want it.

Reply:
Would you not just create the last one from scratch though? It's not really complex enough to warrant a template IMO

Reply:
Yes - exactly! .... The "Create From Scratch" option is THIS template!

I decided just to make everything a template - even "Create from Scratch" - I will just rename to `createFromScratchTemplate.js`